c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -28,6 +29,7 @@
 #include <iostream>
 #include <memory>
 #include <mutex>
+#include <new>
```

Comment:
Since you don't use hardware_destructive_interference_size you don't this one right?